ChangeLog

```diff
@@ -17,6 +17,11 @@ Release date: 2021-04-26
 
 * Pylint's tags are now the standard form ``vX.Y.Z`` and not ``pylint-X.Y.Z`` anymore.
 
+* New warning message ``deprecated-class``. This message is emitted if import or call deprecated class of the
+__standard library (like ``collections.Iterable`` that will be removed in Python 3.10).
```

Comment:
I thought collection.abc was going to stop working in 3.9 ?

Reply:
I checked the cpython code in 3.9[1], ``from collections import Iterable`` still works. And in master(3.10), the function ``__getattr__`` in ``collections.__init__.py`` has been removed.

https://github.com/python/cpython/blob/3.9/Lib/collections/__init__.py#L55